testing/features/support/drivers/browserstack/payload_values_generator.rb

```diff
@@ -35,23 +35,20 @@ def branch?
           docker_tag&.include?("PR-")
         end
 
-        # Example: PR-81
-        def pr_without_build_iteration
-          pr_sha_reference.split(/-\d+_/).first
+        # Example: PR-284
+        def pr_without_sha
+          pr_with_sha.split("_").first
         end
 
-        def build_iteration
-          pr_with_build_iteration.split("-").last
+        # Example: ad4b223
+        def sha
+          pr_with_sha.split("_").last
         end
 
-        # Example: PR-81-1
-        def pr_with_build_iteration
-          pr_sha_reference.split("_").first
-        end
-
-        # Example: PR-81-1_98eb207
-        def pr_sha_reference
-          @pr_sha_reference ||= docker_tag.split("design-system-").last
+        # Example: master_ad4b223
+        # Example: PR-284_dbbb8b6
+        def pr_with_sha
+          @pr_with_sha ||= docker_tag.split("design-system/").last
```

Comment:
I'd write this (and others) as, e.g.

```ruby
@pr_with_sha ||= docker_tag.delete_prefix('design-system/')
```

But it's mostly stylistic as you're only talking about tiny strings.

I'm happy with this change in general.